libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tabset-group.component.scss

```diff
@@ -1,55 +1,111 @@
+  --sky-override-vertical-tabset-group-header-background-color: transparent;
+  --sky-override-vertical-tabset-group-header-open-font-weight: 400;
+  --sky-override-vertical-tabset-group-margin: #{$sky-margin-half} 0 #{$sky-margin-half}
+    #{$sky-margin-half};
+}
+
+@include compatMixins.sky-modern-overrides('.sky-vertical-tabset-group') {
+  --sky-override-tabset-group-expansion-indicator-position: absolute;
+  --sky-override-tabset-group-header-position: relative;
+  --sky-override-vertical-tabset-button-disabled-font-color: buttontext;
+  --sky-override-vertical-tabset-group-button-font-color: buttontext;
+  --sky-override-vertical-tabset-group-gap: #{$sky-theme-modern-space-md};
+  --sky-override-vertical-tabset-group-header-open-font-weight: var(
+    --modern-font-style-regular
+  );
+  --sky-override-vertical-tabset-group-margin: #{$sky-theme-modern-space-md} 0;
+}
 
 .sky-vertical-tabset-group {
-  @include mixins.sky-border(light, bottom);
-  margin: $sky-margin-half 0 $sky-margin-half $sky-margin-half;
+  border-bottom: var(--sky-override-vertical-tabset-group-border, none);
+  margin: var(--sky-override-vertical-tabset-group-margin);
+}
+
+:host:has(~ sky-vertical-tabset-group) .sky-vertical-tabset-group {
+  margin-bottom: var(
+    --sky-override-vertical-tabset-group-gap,
+    var(--sky-space-gap-action_group-l)
+  );
 }
 
 .sky-vertical-tabset-group-header {
+  position: var(--sky-override-tabset-group-header-position, initial);
+
   button {
+    color: var(
+      --sky-override-vertical-tabset-group-button-font-color,
+      var(--sky-color-text-default)
+    );
+    padding: var(
+      --sky-override-vertical-tabset-group-button-padding,
+      var(--sky-space-inset-balanced-s)
+    );
     cursor: pointer;
     display: flex;
     justify-content: space-between;
     border: none;
     width: 100%;
     text-align: left;
-    background: none;
-  }
-}
-
-.sky-vertical-tabset-group-content {
-  overflow: hidden;
-  @include mixins.sky-border(light, top);
-}
-
-:host ::ng-deep .sky-vertical-tabset-group-expansion-indicator > button {
-  font-size: 14px !important;
-}
+    align-items: var(
+      --sky-override-vertical-tabset-group-header-align-items,
+      center
+    );
+    background-color: var(
+      --sky-override-vertical-tabset-group-header-background-color,
+      var(--sky-color-background-nav-base)
+    );
 
-.sky-vertical-tabset-button-disabled {
-  cursor: not-allowed;
-  pointer-events: none;
-}
-
-@include mixins.sky-theme-modern {
-  .sky-vertical-tabset-group {
-    border: none;
-    margin: $sky-theme-modern-space-md 0;
+    &.sky-vertical-tabset-button-disabled {
+      pointer-events: none;
+      color: var(
+        --sky-override-vertical-tabset-button-disabled-font-color,
+        var(--sky-color-text-deemphasized)
+      );
+    }
   }
 
-  .sky-vertical-tabset-group-content {
-    border: none;
-    padding-left: 10px;
+  .sky-vertical-tabset-group-expansion-indicator {
+    position: var(
```

Comment:
The expansion indicator used to be absolutely positioned in modern theme. This maintains that behavior; however, in v2 the expansion indicator will automatically center within the group header vertically. This matches what repeater does and was approved by Matt.